src/extension/ScriptRunner.ts

```diff
@@ -28,24 +29,37 @@ export class ScriptRunner {
   }
 
   private getConfFileName(path: string): string {
-    const splitedPathToConfFile = path.split('/')
-    return splitedPathToConfFile[splitedPathToConfFile.length - 1]
+    const splittedPathToConfFile = path.split('/')
+    return splittedPathToConfFile[splittedPathToConfFile.length - 1]
```

Comment:
Please check the `path` module (or any other module) for doing this.
It may prevent path related issues.

Reply:
when changing conf.log to conf, this part is deleted anyway...